### PR TITLE
Destroy CLoot on collision

### PR DESCRIPTION
**Pull request by Aingar**
CLoot containers are effectively unmovable objects, causing any ship that tries to fly into one to either quickly bounce off in case of smaller ships, or spin violently in case of caps. This destroys the CLoot containers when collided at above-trivial speed (default 25, configurable)

### CODE REVIEW

**Comment by Aingar** [comment]
The added check against ci.dwTargetShip is a performance booster, prevents further execution if we're colliding with clientside debris/asteroids.

File: Plugins/Public/playercntl_plugin/Main.cpp [hunk 1]
```diff
@@ -588,9 +592,28 @@ namespace HkIServerImpl
 		returncode = DEFAULT_RETURNCODE;
 
 		// If spin protection is off, do nothing.
-		if (set_fSpinProtectMass == -1.0f)
+		if (!ci.dwTargetShip || set_fSpinProtectMass == -1.0f)
 			return;
```
Comment (Lazrius) [inline Plugins/Public/playercntl_plugin/Main.cpp:596]: Add braces

**Review by Lazrius — APPROVED** [review]

File: Plugins/Public/playercntl_plugin/Main.cpp (hunk 1 above)
Comment (Aingar) [reply Plugins/Public/playercntl_plugin/Main.cpp:597]: done

**Review by dsyalex — APPROVED** [review]
Per Laz's approval, extra diff since looks good

**Pull request merged**